src/stringLengthUtf8.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Returns the number of bytes that would be used when encoding the string as UTF-8, effectively the
+ * same as `new TextEncoder().encode(str).length` but faster.
```

Comment:
worth explaining why it's faster or what the tradeoff for speed is?

Reply:
I'll add a link to the jsbench
```suggestion
 * same as `new TextEncoder().encode(str).length` but faster.
 * https://jsbench.me/nzlrkwmeiq/1
```